recipes/zxing-cpp/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "2.0.0":
+    url: "https://github.com/nu-book/zxing-cpp/archive/v1.4.0.tar.gz"
+    sha256: "126767bb56f8a1f25ae84d233db2e9b9be50d71f5776092d0e170ca0f0ed1862"
```

Comment:
That is the 1.4 url and hash, not the 2.0 one.

Reply:
RIP, please open a new PR and tag me 🙏